frontend/test/playwright/e2e/search-query-server.spec.ts

```diff
@@ -91,21 +91,13 @@ test.describe("search query on SSR", () => {
       }
     })
 
-    test.skip("url mature query is set, and can be unchecked using the Safer Browsing popup", async ({
-      page,
-    }) => {
-      await goToSearchTerm(page, "cat", {
-        searchType: IMAGE,
-        query: "mature=true",
-      })
-
-      await page.click('button:has-text("Safer Browsing")')
-
-      const matureCheckbox = await page.locator("text=Show Mature Content")
-      await expect(matureCheckbox).toBeChecked()
-
-      await page.click("text=Show Mature Content")
-      await expect(page).toHaveURL("/search/image?q=cat")
-    })
+    test.fixme(
```

Comment:
Now I kind of think we should just remove it. It doesn't really have any meaning now.

Reply:
We could keep some of the implementation, if that is a more compelling reason to keep the test. I'm also fine with just deleting it.